gradle.properties

```diff
@@ -56,3 +56,6 @@ dunitParallelForks = 8
 dunitDockerImage = apachegeode/geode-build
 # Docker user for parallel dunit tests
 dunitDockerUser = root
+
+#JVM to be used by tests
+#testJVM=
```

Comment:
Is this left commented out as a template for how the test JVM might be overridden?

Reply:
I'm not sure.   That was done by someone else I think - Galen or Dan.  I'm guessing we don't have support for actually pulling down a different JDK for CI runs at this point.